dask_sql/physical/rex/core/call.py

```diff
@@ -812,11 +853,11 @@ def date_part(self, what, df: SeriesOrScalar):
             return df.dayofyear
         elif what == "HOUR":
             return df.hour
-        elif what == "MICROSECOND":
+        elif what in {"MICROSECOND", "MICROSECONDS"}:
             return df.microsecond
-        elif what == "MILLENNIUM":
+        elif what in {"MILLENIUM", "MILLENNIUM"}:
```

Comment:
Is `millenium` supported in specific grammars? I assumed it was a spelling variant of millennium but it seems like it's a misspelling?

Reply:
I mainly added it because originally sqlparser-rs only supported `millenium` until support for both spellings was added with https://github.com/sqlparser-rs/sqlparser-rs/pull/633. I could change it back to just `millennium` though if desired.

Reply:
Thanks for the context! In that case, I'm okay with leaving both spellings in 🙂